src/components/Trade/hooks/useFiatRateService.tsx

```diff
@@ -24,7 +24,7 @@ export const useFiatRateService = () => {
   const [usdRatesArgs, setUsdRatesArgs] = useState<UsdRatesInputArgs>(skipToken)
 
   // Selectors
-  const selectedCurrencyToUsdRate = useAppSelector(selectFiatToUsdRate)
+  const selectedCurrencyToUsdRate = useAppSelector(selectFiatToUsdRate).toString()
```

Comment:
stringify in selector perhaps, unless there's a reason for this?

Reply:
No reason, good catch: addressed in https://github.com/shapeshift/web/pull/4138/commits/dddbde80b84b26edfea966547554a69f2d10f290.